plugins/Goals/lang/en.json

```
@@ -3,63 +3,63 @@
         "AbandonedCart": "Abandoned Cart",
         "AddGoal": "Add Goal",
         "AddNewGoal": "Add a new Goal",
-        "AddNewGoalOrEditExistingGoal": "%1$sAdd a new Goal%2$s or %3$sEdit%4$s existing Goals",
-        "AllowGoalConvertedMoreThanOncePerVisit": "Allow Goal to be converted more than once per visit",
+        "AddNewGoalOrEditExistingGoal": "%1$sAdd a new goal%2$s or %3$sEdit%4$s existing goals",
+        "AllowGoalConvertedMoreThanOncePerVisit": "Allow goal to be converted more than once per visit",
```

```suggestion
        "AllowGoalConvertedMoreThanOncePerVisit": "Allow any goal to be converted more than once per visit",
```
Can also be "a", "the", "goals", or "this". Small improvement.